connector/src/yang/connector/netconf.py

```diff
@@ -210,6 +216,18 @@ def __init__(self, *args, **kwargs):
             self, session=session, device_handler=device_handler,
             timeout=self.timeout)
 
+    active_notifications = {}
+
+    def subscribe(self, request, steps):
+        subscribe_thread = NetconfNotification(self, request=request)
+
+        if not self.connected:
+            self.connect()
+
+        subscribe_thread.start()
+
+        self.active_notifications[self] = subscribe_thread
```

Comment:
I'm not convinced to use `self` as a key of `self.active_notifications`. For one Netconf session, how many `subscribe_thread` are allowed?

Reply:
should I use a unique ID for the key of `self.active_notifications`? Currently for one NETCONF session, there is no limit on how many `subscribe_thread` are allowed.

Reply:
I used a unique id to reference the notification thread